test/integration/check_navbar_links.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Checks that the links in the navbar on the homepage have the right URLs to redirect to
+ *
+ * Test cases: https://github.com/LLK/scratch-www/wiki/Most-Important-Workflows#Create_should_take_you_to_the_editor
+ */
+
+var tap=require('tap'),
+    seleniumWebdriver = require('selenium-webdriver');
+
+/*
+ * Remove question comments after resolving them
+ */
+//how to generalize for other browsers... how will this work in saucelabs? Do I need to iterate 
+//through the drivers for each browser here?
+//chrome driver
+var driver = new seleniumWebdriver.Builder().withCapabilities(seleniumWebdriver.Capabilities.chrome()).build();
+//open scratch.ly in a new instance of the browser
+driver.get('https://scratch.ly');
+
+/*
+ * Remove question comments after resolving them
+ */
+//find the navbar
+//var navbarElement = driver.findElement(seleniumWebdriver.By.id("navigation"))
+//var createLinkSignedOut = navbarElement.findElement(seleniumWebdriver.By.xpath('//li[@class="link create"]/a'))
+//^^this doesn't work to force findElement to look in the scope of navbarElement
+
+//find the create link within the navbar
+//the create link depends on whether the user is signed in or not (tips window opens)
+
+/*
+ * Remove question comments after resolving them
+ */
+//this xpath is fragile, can i look up by successive attributes instead?
```

Comment:
I think using the fragile XPath is a necessary evil of using Selenium this way. When we change the DOM structure, we will have to update the test, but this is frequently the case with refactors even when not using Selenium.